PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/StringSequenceNode.java

```diff
@@ -0,0 +1,165 @@
+package nl.tudelft.pl2016gr2.model;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+/**
+ * A simple implementation of <code>Node</code> that represents the DNA sequence as String.
+ *
+ * @author Wouter Smit
+ */
+public class StringSequenceNode extends AbstractNode {
+
+  private String sequence;
+  private HashSet<String> genomes;
+  private HashSet<Integer> inEdges;
+  private HashSet<Integer> outEdges;
```

Comment:
`contains()` is actuallly called quite a few times in the code to inspect the neighbours of a node. However, if the assumptation that every node has only few edges holds (and I do think it does), the lesser performance of `contains()` in a non-`HashTable` is not really a downside. I will implement this.